project/ws/viewer/src/lib/viewer-util.service.ts

```diff
@@ -17,6 +17,8 @@ export class ViewerUtilService {
   }
   downloadRegex = new RegExp(`(/content-store/.*?)(\\\)?\\\\?['"])`, 'gm')
   authoringBase = '/apis/authContent/'
+  competencyAsessment = new BehaviorSubject<any>(null)
```

Comment:
new BehaviorSubject<any>(null) change this one to new BehaviorSubject<boolean>(false);